### PR TITLE
Adding comments for MQTT streams and Jobs library functions.

### PR DESCRIPTION
**Pull request by manvensh**
Adding comments where MQTT streams and Jobs library functions have been called.

### CODE REVIEW

File: demo/ota-Agent-Orchestrator/ota_demo.c [hunk 1]
```diff
@@ -177,7 +186,12 @@ static void initMqttDownloader( AfrOtaJobDocumentFields_t *jobFields )
 
     mqttWrapper_getThingName( thingName, &thingNameLength );
 
-    /* Initialize the File downloader */
+    /*
+     * MQTT streams Library:
+     * Initializing the MQTT streams downloader. Passing the
+     * paramters extracted from the AWS IoT OTA jobs document
```
Comment (bradleysmith23) [inline demo/ota-Agent-Orchestrator/ota_demo.c:192]: parameters

File: demo/simple-Ota-Orchestrator/ota_demo.c [hunk 2]
```diff
@@ -96,13 +111,23 @@ bool otaDemo_handleIncomingMQTTMessage( char * topic,
         }
         else
         {
+            /*
+             * MQTT streams Library:
+             * Checks if the incoming message contains the requested data block. It is perfromed by
```
Comment (bradleysmith23) [inline demo/simple-Ota-Orchestrator/ota_demo.c:116]: performed

File: demo/simple-Ota-Orchestrator/ota_demo.c [hunk 3]
```diff
@@ -247,7 +292,12 @@ static void processJobFile( AfrOtaJobDocumentFields_t * params )
     currentFileId = params->fileId;
     currentBlockOffset = 0;
     totalBytesReceived = 0;
-    /* Initialize the File downloader */
+    /*
+     * MQTT streams Library:
+     * Initializing the MQTT streams downloader. Passing the
+     * paramters extracted from the AWS IoT OTA jobs document
```
Comment (bradleysmith23) [inline demo/simple-Ota-Orchestrator/ota_demo.c:298]: parameters

**Review by bradleysmith23 — COMMENTED** [review]
Looks good, just fix the few spelling errors.

File: demo/simple-Ota-Orchestrator/ota_demo.c (hunk 3 above)
Comment (manvensh) [reply demo/simple-Ota-Orchestrator/ota_demo.c]: Fixed in the latest commit

File: demo/ota-Agent-Orchestrator/ota_demo.c (hunk 1 above)
Comment (manvensh) [reply demo/ota-Agent-Orchestrator/ota_demo.c]: Fixed in the latest commit

File: demo/simple-Ota-Orchestrator/ota_demo.c (hunk 2 above)
Comment (manvensh) [reply demo/simple-Ota-Orchestrator/ota_demo.c]: Fixed in the latest commit

**Review by bradleysmith23 — APPROVED** [review]

**Pull request merged**